scripts/scrapeEspn.py

```
@@ -0,0 +1,98 @@
+        return result
+    except Exception as e:
+        print(f"Error saving player {player['name']}: {str(e)}")
+        return None
+
+def main():
+    sports = ['nba', 'mlb', 'nfl', 'boxing']
+    for sport in sports:
+        print(f"\nScraping {sport.upper()} players...")
+        players = get_player_list(sport)
+        print(f"Found {len(players)} players")
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)